helm-chart/ray-cluster/values.yaml

```
@@ -211,6 +214,9 @@ worker:
   serviceAccountName: ""
   restartPolicy: ""
 
+  # runtimeClassName is the name of the RuntimeClass to use to run the worker Pods.
```

```suggestion
  # -- runtimeClassName is the name of the RuntimeClass to use to run the worker Pods.
```
For [helm-docs](https://github.com/norwoodj/helm-docs) to automatically generate values table in Helm chart README.